src/components/expression/expression-loader.js

```diff
@@ -170,6 +169,22 @@ genotet.ExpressionLoader.prototype.loadTfaProfile_ = function(fileName,
 };
 
 /**
+ * Loads expression data list into genotet.data.expressionFiles.
```

Comment:
Since this function is writing data to genotet.data, it more likely should belong to genotet.data. Otherwise, there could be multiple genotet.ExpressionLoaders loading into the same genotet.data.expressionFiles object, which is not a good design pattern.